app/config/routes.rb

```diff
@@ -42,9 +42,12 @@
     # (remove this once people know about the new invitation URL)
     get "/invitations/new", to: redirect { |_, req| "/nyc/invitations/new?secret=#{req.params[:secret]}" }
 
-    scope "/:site_id", module: :caseworker do
-      get "/sso/", to: "sso#index", as: :new_user_session
-      resources :cbv_flow_invitations, as: :invitations, path: :invitations
+    constraints(site_id: Regexp.union(Rails.application.config.sites.site_ids)) do
+      scope "/:site_id", module: :caseworker do
```

Comment:
I think putting it on the `scope` itself is a bit more intuitive - since it's referring to the "site_id" parameter that is specified by `scope "/:site_id"` if you see what I mean? It's kind of weird to have the constraint at a higher level than the scope that defines the variable.

```suggestion
      scope "/:site_id", module: :caseworker, constraints: { site_id: Regexp.union(Rails.application.config.sites.site_ids) } do
```